src/api/financial/controllers/currency-conversion.ts

```diff
@@ -2,6 +2,8 @@
  *  controller
  */
 
-import { factories } from '@strapi/strapi'
+import { factories } from "@strapi/strapi";
 
-export default factories.createCoreController('api::financial.currency-conversion');
+export default factories.createCoreController(
+  "api::financial.currency-conversion",
```

Comment:
**question(non-blocking):** Why'd the formatter decide to add a comma?  Does that indicate there are other optional params?  It doesn't do this in other controllers.

Reply:
Once it's broken onto its own line it adds a trailing comma. Trailing commas are nice because they reduce git churn. If someone came in here later and started passing a second value to this function they wouldn't need to change this line since the comma is already there.